libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -165,7 +168,7 @@ static bool _mqttSubscription_setUnsubscribe( const IotLink_t * pSubscriptionLin
     return true;
 }
 
-/*-----------------------------------------------------------*/
+/*------------------------------------------------------------------------------*/
```

Comment:
Nit: I'm not sure if you intentionally had this line to be longer than the others but it is really no biggie